sphinx/writers/latex.py

```diff
@@ -841,10 +841,15 @@ def depart_rubric(self, node: Element) -> None:
     def visit_footnote(self, node: Element) -> None:
         self.in_footnote += 1
         label = cast(nodes.label, node[0])
+        if 'auto' not in node.attributes:
+            self.body.append('\\sphinxstepexplicit ')
         if self.in_parsed_literal:
             self.body.append('\\begin{footnote}[%s]' % label.astext())
         else:
             self.body.append('%%\n\\begin{footnote}[%s]' % label.astext())
+        if 'auto' not in node.attributes:
```

Comment:
docutils provides a shortcut to check the existence of an attribute: `in` and `not in`. So you don't need to use "attributes" attribute.
```suggestion
        if 'auto' not in node:
```